python/hail/representation/pedigree.py

```diff
@@ -0,0 +1,321 @@
+from hail.java import *
+
+
+class Trio(object):
+    """Class containing information about nuclear family relatedness and sex.
+
+    :param str proband: sample ID of proband
+
+    :param father: sample ID of proband
+    :type father: str or None
+
+    :param mother: sample ID of proband
+    :type mother: str or None
+
+    :param sex: sex of proband: 1 for male, 2 for female
```

Comment:
I'm not sure I see this.  What can you write with ints here that isn't as natural to write with booleans?